src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/project.json

```diff
@@ -23,7 +23,7 @@
     }
   },
   "dependencies": {
-    "Microsoft.DotNet.InternalAbstractions": "1.0.0",
+    "Microsoft.DotNet.InternalAbstractions": "1.0.1-*",
```

Comment:
Get this dependency conflict otherwise:
C:\dev\scaffolding\src\Microsoft.VisualStudio.Web.CodeGeneration.Tools\project.json(26,52): warning NU1012: Dependency c
onflict. Microsoft.Extensions.DependencyModel 1.0.1-beta-003206 expected Microsoft.DotNet.InternalAbstractions >= 1.0.1-
beta-003206 but received 1.0.0